sdk/storage/azure-storage-file-datalake/swagger/src/main/java/com/azure/storage/file/datalake/customization/DataLakeStorageCustomization.java

```diff
@@ -8,13 +8,14 @@
 import com.azure.autorest.customization.LibraryCustomization;
 import com.azure.autorest.customization.MethodCustomization;
 import com.azure.autorest.customization.PackageCustomization;
+import org.slf4j.Logger;
 
 /**
  * Customization class for File DataLake Storage.
  */
 public class DataLakeStorageCustomization extends Customization {
     @Override
-    public void customize(LibraryCustomization customization) {
+    public void customize(LibraryCustomization customization, Logger logger) {
```

Comment:
What is this?

Reply:
Autorest had a breaking change in their interface. This was a necessary change to generate new code